bobtemplates/plone_addon/buildout.cfg.bob

```diff
@@ -28,6 +28,8 @@ flake8-extensions =
     flake8-blind-except
     flake8-debugger
     flake8-coding
+return-status-codes = True
```

Comment:
This should go into travis.cfg IMHO. We do not want to return status codes locally.